README.md

```diff
@@ -330,4 +331,242 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
+```
+
+### 8. Pull secrets
+
+Go to the config.json file in "C:\Users\yourUserProfile\.docker", and delete anything inside the string after credsStore to make sure that it is empty as follows:
+
+```bash
+"credsStore": ""
+```
+
+
+Log in to quay.io.
+```bash
+docker login quay.io
+```
+Give your username and password if prompted.
+
+Create a variable that stores an encoded version of your docker-config file. 
+
+
+```bash
+$dockerConfigFile = GET-CONTENT -Path "$env:USERPROFILE\.docker\config.json"
+$QUAY_SECRET =[Convert]::ToBase64String([System.Text.Encoding]::ASCII.GetBytes($dockerConfigFile))
+```
+
+Create a secret.yaml file using the $QUAY_SECRET variable.
+
+```bash
+echo "apiVersion: v1
+kind: Secret
+metadata:
+  name: quay-registry-secret
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: $QUAY_SECRET" > secrets.yaml
+``` 
+ 
+
+Create the secret in your namespace. 
+
+```bash
+kubectl create -f secrets.yaml --namespace $DESIREDNAMESPACE
+```
+
+### 9. Add remote chart repository to Helm configuration.
+
+```bash
+helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+```
+
+### 10. Authorize connections
+
+Go back to the config.json file, and check that there is a string after "auth", such as in the following example.
+
+```bash
+"auth": "klsdjfsdkifdsiEWRFJDOFfslakfdjsidjfdslfjds"
+```
+
+Do the following command to find your ipv4 address and copy it to your clipboard.
+
+```bash
+ipconfig
+```
+
+Note: If there are many ipv4 addresses, use the ipv4 address of the connection that you are currently using.
+
+
+Paste the ipv4 address at the end of the hosts file in C:\Windows\System32\drivers\etc, followed by "localhost-k8s" as in the following example. 
+
+```bash
+# Copyright (c) 1993-2009 Microsoft Corp.
+#
+# This is a sample HOSTS file used by Microsoft TCP/IP for Windows.
+#
+# This file contains the mappings of IP addresses to host names. Each
+# entry should be kept on an individual line. The IP address should
+# be placed in the first column followed by the corresponding host name.
+# The IP address and the host name should be separated by at least one
+# space.
+#
+# Additionally, comments (such as these) may be inserted on individual
+# lines or following the machine name denoted by a '#' symbol.
+#
+# For example:
+#
+#      102.54.94.97     rhino.acme.com          # source server
+#       38.25.63.10     x.acme.com              # x client host
+
+# localhost name resolution is handled within DNS itself.
+#	127.0.0.1       localhost
+#	::1             localhost
+
+# Added by Docker for Windows
+10.244.50.193 host.docker.internal
+10.244.50.193 gateway.docker.internal
+# End of section
+
+<Your ipv4 address> localhost-k8s
+
+```
+
+Note: Make sure to leave a new line at the end before saving it. 
+
+### 11. Install alfresco-dbp
+
+Copy and paste the following block into your command line.
+  
+```bash
+helm install alfresco-incubator/alfresco-dbp `
```

Comment:
Shouldn't this be updated to pull in `minimal-values.yaml` now?

Reply:
exactly :)
i Just figured it our after merging and reverted.